src/app/cli/src/.ocamlformat

```diff
@@ -0,0 +1,66 @@
+profile=ocamlformat
```

Comment:
Can you make this file a link to `src/.ocamlformat`? It would be good to know that updating that one will also update here automatically.

Reply:
done